docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -25,9 +25,17 @@ The following options can optionally be configured:
 
 {{< schema root="BuildpackArtifact" >}}
 
+**Builder**
+
 `builder` is *required* and tells Skaffold which
 [Builder](https://buildpacks.io/docs/app-developer-guide/build-an-app/) to use.
 
+**User defined environment variables**
+
+`env` makes it possible to pass arbitrary environment variables to the buildpacks.
+This is in general used to adjust the detection and the build phases. Many buildpacks
+use environment variables to select specific versions of language runtimes.
+
 **Example**
 
 The following `build` section, instructs Skaffold to build a
```

Comment:
Could you also add an example of `env` — maybe with a go-template? — to the `samples/builders/buildpacks.yaml`?  (Is it possible to expose a value from the user's environment with this mechanism? `USER={{.USER}}`?)

(I likely don't know what I'm talking about here: I know only enough go templating to process `docker images —format`.)

Reply:
That's a good idea to add such value to the sample. But I don't see any value that would really be useful. Any idea?

Reply:
Sorry, with regards to nested templates, I still don't understand what your point is.

Reply:
`NPM_CONFIG_LOGLEVEL`?

Reply:
Add GOPROXY